python/foglamp/services/core/api/service.py

```diff
@@ -6,24 +6,56 @@
 
 import datetime
 from aiohttp import web
+from foglamp.services.core.service_registry.service_registry import ServiceRegistry
 from foglamp.common.storage_client.payload_builder import PayloadBuilder
 from foglamp.common.configuration_manager import ConfigurationManager
 from foglamp.services.core import server
 from foglamp.services.core import connect
 from foglamp.services.core.scheduler.entities import StartUpSchedule
 
-__author__ = "Mark Riddoch"
+__author__ = "Mark Riddoch, Ashwin Gopalakrishnan"
 __copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
 _help = """
     -------------------------------------------------------------------------------
     | POST            | /foglamp/service                                          |
+    | GET             | /foglamp/service                                          |
     -------------------------------------------------------------------------------
 """
 
 
+#################################
+#  Service
+#################################
+
+
+async def get_health(request):
+    """
+    Args:
+        request:
+
+    Returns:
+            health of all registered services
+
+    :Example:
+            curl -X GET http://localhost:8081/foglamp/service
+    """
+    sr_list = list()
+    for service_record in ServiceRegistry.all():
+        sr_list.append(
+                {
+                    'name' : service_record._name,
+                    'type' : service_record._type,
+                    'address' : service_record._address,
+                    'management_port' : service_record._management_port,
+                    'service_port' : service_record._port,
+                    'protocol' : service_record._protocol,
+                    'status': 'running'
```

Comment:
In the future this status will come from the monitored state in the service registry. However for now we only have an entry for running services, therefore this is a reasonable first implementation.